core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -158,8 +158,9 @@ var ExplorationPlayerPage = function() {
     });
   };
 
-  //`answerData` is a variable that is passed to the corresponding interaction's
-  //protractor utilities. Its definition and type are interaction-specific.
+  // `answerData` is a variable that is passed to the
+  //  corresponding interaction's protractor utilities.
```

Comment:
remove one space in the beginning, so that `corresponding` lines up  with the backtick from the previous line

Reply:
My bad! Done, thanks!